ocp_resources/cluster_role_binding.py

```diff
@@ -9,3 +10,27 @@ class ClusterRoleBinding(Resource):
     """
 
     api_group = Resource.ApiGroup.RBAC_AUTHORIZATION_K8S_IO
+
+    def __init__(
+        self,
+        name=None,
+        cluster_role=None,
+        subjects=None,
+    ):
+        super().__init__(name=name)
+        self.cluster_role = cluster_role
+        self.subjects = subjects
+        self.res = None
```

Comment:
```
        self.res = None
```
Why this is needed?

Reply:
To allow adding rules after initiating object